2023-10-10 14:19:21,795 DEBUG    root            Loaded Command Group: ['gcloud', 'compute']
2023-10-10 14:19:21,796 DEBUG    root            Loaded Command Group: ['gcloud', 'compute', 'instances']
2023-10-10 14:19:21,904 DEBUG    root            Loaded Command Group: ['gcloud', 'compute', 'instances', 'create']
2023-10-10 14:19:21,979 DEBUG    root            Running [gcloud.compute.instances.create] with arguments: [--boot-disk-size: "536870912000", --image-family: "tf-1-15-5", --image-project: "ml-images", --machine-type: "n1-highmem-8", --maintenance-policy: "TERMINATE", --restart-on-failure: "True", --zone: "asia-southeast1-b", INSTANCE_NAMES:1: "['pegasus']"]
2023-10-10 14:19:21,990 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): compute.googleapis.com:443
2023-10-10 14:19:22,511 DEBUG    urllib3.connectionpool https://compute.googleapis.com:443 "GET /compute/v1/projects/pegasus-401519/zones/asia-southeast1-b?alt=json HTTP/1.1" 200 None
2023-10-10 14:19:22,518 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): compute.googleapis.com:443
2023-10-10 14:19:23,066 DEBUG    urllib3.connectionpool https://compute.googleapis.com:443 "GET /compute/v1/projects/pegasus-401519?alt=json HTTP/1.1" 200 None
2023-10-10 14:19:23,080 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): compute.googleapis.com:443
2023-10-10 14:19:25,646 DEBUG    urllib3.connectionpool https://compute.googleapis.com:443 "POST /compute/v1/projects/pegasus-401519/zones/asia-southeast1-b/instances?alt=json HTTP/1.1" 200 None
2023-10-10 14:19:25,648 DEBUG    root            Operations to inspect: [(<Operation
 id: 1871811385705934586
 insertTime: '2023-10-10T01:49:25.452-07:00'
 kind: 'compute#operation'
 name: 'operation-1696927763176-60758c8889a79-f20e64f4-8862f3c4'
 operationType: 'insert'
 progress: 0
 selfLink: 'https://www.googleapis.com/compute/v1/projects/pegasus-401519/zones/asia-southeast1-b/operations/operation-1696927763176-60758c8889a79-f20e64f4-8862f3c4'
 startTime: '2023-10-10T01:49:25.452-07:00'
 status: StatusValueValuesEnum(RUNNING, 2)
 targetId: 8904997465947156219
 targetLink: 'https://www.googleapis.com/compute/v1/projects/pegasus-401519/zones/asia-southeast1-b/instances/pegasus'
 user: 'hardiksharmah73@gmail.com'
 warnings: []
 zone: 'https://www.googleapis.com/compute/v1/projects/pegasus-401519/zones/asia-southeast1-b'>, 3)]
2023-10-10 14:19:25,649 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): compute.googleapis.com:443
2023-10-10 14:19:31,450 DEBUG    urllib3.connectionpool https://compute.googleapis.com:443 "POST /compute/v1/projects/pegasus-401519/zones/asia-southeast1-b/operations/operation-1696927763176-60758c8889a79-f20e64f4-8862f3c4/wait?alt=json HTTP/1.1" 200 None
2023-10-10 14:19:31,453 DEBUG    root            Operations to inspect: [(<Operation
 endTime: '2023-10-10T01:49:31.341-07:00'
 error: <ErrorValue
 errors: [<ErrorsValueListEntry
 code: 'ZONE_RESOURCE_POOL_EXHAUSTED'
 errorDetails: [<ErrorDetailsValueListEntry
 help: <Help
 links: [<HelpLink
 description: 'Troubleshooting documentation'
 url: 'https://cloud.google.com/compute/docs/resource-error'>]>>, <ErrorDetailsValueListEntry
 localizedMessage: <LocalizedMessage
 locale: 'en-US'
 message: 'A n1-highmem-8 VM instance is currently unavailable in the asia-southeast1-b zone. Alternatively, you can try your request again with a different VM hardware configuration or at a later time. For more information, see the troubleshooting documentation.'>>, <ErrorDetailsValueListEntry
 errorInfo: <ErrorInfo
 domain: 'compute.googleapis.com'
 metadatas: <MetadatasValue
 additionalProperties: [<AdditionalProperty
 key: 'zone'
 value: 'asia-southeast1-b'>, <AdditionalProperty
 key: 'attachment'
 value: ''>, <AdditionalProperty
 key: 'vmType'
 value: 'n1-highmem-8'>, <AdditionalProperty
 key: 'zonesAvailable'
 value: ''>]>
 reason: 'resource_availability'>>]
 message: "The zone 'projects/pegasus-401519/zones/asia-southeast1-b' does not have enough resources available to fulfill the request.  Try a different zone, or try again later.">]>
 httpErrorMessage: 'SERVICE UNAVAILABLE'
 httpErrorStatusCode: 503
 id: 1871811385705934586
 insertTime: '2023-10-10T01:49:25.452-07:00'
 kind: 'compute#operation'
 name: 'operation-1696927763176-60758c8889a79-f20e64f4-8862f3c4'
 operationType: 'insert'
 progress: 100
 selfLink: 'https://www.googleapis.com/compute/v1/projects/pegasus-401519/zones/asia-southeast1-b/operations/operation-1696927763176-60758c8889a79-f20e64f4-8862f3c4'
 startTime: '2023-10-10T01:49:25.452-07:00'
 status: StatusValueValuesEnum(DONE, 0)
 targetId: 8904997465947156219
 targetLink: 'https://www.googleapis.com/compute/v1/projects/pegasus-401519/zones/asia-southeast1-b/instances/pegasus'
 user: 'hardiksharmah73@gmail.com'
 warnings: []
 zone: 'https://www.googleapis.com/compute/v1/projects/pegasus-401519/zones/asia-southeast1-b'>, 3)]
2023-10-10 14:19:31,454 DEBUG    root            Chosen display Format:yaml
2023-10-10 14:19:31,459 DEBUG    root            (gcloud.compute.instances.create) Could not fetch resource:
---
code: ZONE_RESOURCE_POOL_EXHAUSTED
errorDetails:
- help:
    links:
    - description: Troubleshooting documentation
      url: https://cloud.google.com/compute/docs/resource-error
- localizedMessage:
    locale: en-US
    message: A n1-highmem-8 VM instance is currently unavailable in the asia-southeast1-b
      zone. Alternatively, you can try your request again with a different VM hardware
      configuration or at a later time. For more information, see the troubleshooting
      documentation.
- errorInfo:
    domain: compute.googleapis.com
    metadatas:
      attachment: ''
      vmType: n1-highmem-8
      zone: asia-southeast1-b
      zonesAvailable: ''
    reason: resource_availability
message: The zone 'projects/pegasus-401519/zones/asia-southeast1-b' does not have
  enough resources available to fulfill the request.  Try a different zone, or try
  again later.
Traceback (most recent call last):
  File "/home/devilunraveled/GoogleCloud/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 987, in Execute
    resources = calliope_command.Run(cli=self, args=args)
  File "/home/devilunraveled/GoogleCloud/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 807, in Run
    resources = command_instance.Run(args)
  File "/home/devilunraveled/GoogleCloud/google-cloud-sdk/lib/surface/compute/instances/create.py", line 699, in Run
    return compute_client.MakeRequests(requests)
  File "/home/devilunraveled/GoogleCloud/google-cloud-sdk/lib/googlecloudsdk/api_lib/compute/client_adapter.py", line 156, in MakeRequests
    utils.RaiseToolException(
  File "/home/devilunraveled/GoogleCloud/google-cloud-sdk/lib/googlecloudsdk/api_lib/compute/utils.py", line 139, in RaiseToolException
    RaiseException(problems, calliope_exceptions.ToolException, error_message)
  File "/home/devilunraveled/GoogleCloud/google-cloud-sdk/lib/googlecloudsdk/api_lib/compute/utils.py", line 148, in RaiseException
    raise exception(
googlecloudsdk.calliope.exceptions.ToolException: Could not fetch resource:
---
code: ZONE_RESOURCE_POOL_EXHAUSTED
errorDetails:
- help:
    links:
    - description: Troubleshooting documentation
      url: https://cloud.google.com/compute/docs/resource-error
- localizedMessage:
    locale: en-US
    message: A n1-highmem-8 VM instance is currently unavailable in the asia-southeast1-b
      zone. Alternatively, you can try your request again with a different VM hardware
      configuration or at a later time. For more information, see the troubleshooting
      documentation.
- errorInfo:
    domain: compute.googleapis.com
    metadatas:
      attachment: ''
      vmType: n1-highmem-8
      zone: asia-southeast1-b
      zonesAvailable: ''
    reason: resource_availability
message: The zone 'projects/pegasus-401519/zones/asia-southeast1-b' does not have
  enough resources available to fulfill the request.  Try a different zone, or try
  again later.

2023-10-10 14:19:31,461 ERROR    root            (gcloud.compute.instances.create) Could not fetch resource:
---
code: ZONE_RESOURCE_POOL_EXHAUSTED
errorDetails:
- help:
    links:
    - description: Troubleshooting documentation
      url: https://cloud.google.com/compute/docs/resource-error
- localizedMessage:
    locale: en-US
    message: A n1-highmem-8 VM instance is currently unavailable in the asia-southeast1-b
      zone. Alternatively, you can try your request again with a different VM hardware
      configuration or at a later time. For more information, see the troubleshooting
      documentation.
- errorInfo:
    domain: compute.googleapis.com
    metadatas:
      attachment: ''
      vmType: n1-highmem-8
      zone: asia-southeast1-b
      zonesAvailable: ''
    reason: resource_availability
message: The zone 'projects/pegasus-401519/zones/asia-southeast1-b' does not have
  enough resources available to fulfill the request.  Try a different zone, or try
  again later.

